cfme/tests/cloud_infra_common/test_rest_providers.py

```diff
@@ -91,8 +91,13 @@ def provider_rest(request, appliance, provider):
     else:
         pytest.skip("No credentials info found for provider {}.".format(provider.name))
 
-    if hasattr(endpoint_default, "verify_tls") and not endpoint_default.verify_tls:
-        default_connection["endpoint"]["verify_ssl"] = 0
+    cert = getattr(endpoint_default, "ca_certs", None)
+    if cert:
+        default_connection["endpoint"]["certificate_authority"] = cert
+        con_config_to_include.append(default_connection)
+
+    if hasattr(endpoint_default, "verify_tls"):
+        default_connection["endpoint"]["verify_ssl"] = 1 if endpoint_default.verify_tls else 0
```

Comment:
**Required**
Use True/False instead of 1/0 if possible here.

Reply:
In fact I'm pretty sure you can just write this as:

```python
default_connection['endpoint']['verify_ssl'] = endpoint_default.verify_tls

Reply:
@mshriver, these are not form values but dictionary representing JSON of REST request. The values needs to be 0 or 1, not True or False.

Reply:
_fine_ ;)